src/bot/helpers/grammy-context.ts

```diff
@@ -7,39 +7,53 @@ import { Context as UbiquityOsContext } from "../../types";
 import { Logger } from "../../utils/logger";
 import { createAdapters } from "../../adapters";
 import { PluginContext } from "../../types/plugin-context-single";
+import { Octokit as AppOctokit } from "octokit";
```

Comment:
Why as AppOctokit?

Reply:
Theres `octokit` and there's `@octokit/rest` which offer two different typed Octokits. As far as I can tell nothing that affects how we use it.

Reply:
That doesnt answer my question. Call it restOctokit maybe

Reply:
normally we go `new Octokit({auth:"key"})`.

to get appOctokit you new to do `new App({private_key:"", app_id:0}).octokit`, `App` is exported from the `octokit` package.

they both offer rest, gql all the apis. I'm sure it's that `App.octokit` has plugins that the standard octokit does which makes the types different. Same octokit underneath.

Will change to `restOctokit` no probs

Reply:
Changed to `RestOctokitFromApp`